test/integration/Adapter/Driver/Mysqli/TraitSetup.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-db for the canonical source repository
+ * @copyright Copyright (c) 2005-2018 Zend Technologies USA Inc. (https://www.zend.com)
+ * @license   https://github.com/zendframework/zend-db/blob/master/LICENSE.md New BSD License
+ */
+
+namespace ZendIntegrationTest\Db\Adapter\Driver\Mysqli;
+
+trait TraitSetup
```

Comment:
Let's please not use traits for stashing out duplicated code: this is static code, and should be kept as such (static utility)

Reply:
Can you elaborate why? Moreover, you mentioned this is static code?! Thanks.